service/build.gradle

```diff
@@ -5,7 +5,8 @@ dependencies {
     api project(":client")
     api 'com.squareup.retrofit2:retrofit:2.9.0'
     implementation 'com.squareup.retrofit2:adapter-rxjava2:2.9.0'
-    implementation 'com.squareup.retrofit2:converter-jackson:2.9.0'
+    implementation 'com.squareup.retrofit2:converter-jackson:2.7.2'
+    implementation 'com.kjetland:mbknor-jackson-jsonschema_2.13:1.0.39'
```

Comment:
?

Reply:
This build of com.kjetland:mbknor-jackson-jsonschema_2.13:1.0.39 is very vulnerable and sus?
Why would you downgrade the jackson converter to 2.7.2?
https://mvnrepository.com/artifact/com.kjetland/mbknor-jackson-jsonschema_2.13/1.0.39